german/schwimmen/schwimmen.md

```diff
@@ -4,6 +4,8 @@ Sie erreichen Sie mit letzter Kraft und erholen sich erst einmal am Strand. Doch
 
 Man sollte ...
 
+... [etwas scharfes finden, vielleicht einen Stein, und diese Fischerleine aus den Flügeln kriegen.](fluegel_freischneiden/fluegel_freischneiden.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.